Libraries/Image/Image.ios.js

```diff
@@ -62,7 +62,7 @@ var Image = React.createClass({
     /**
      * `uri` is a string representing the resource identifier for the image, which
      * could be an http address, a local file path, or the name of a static image
-     * resource (which should be wrapped in the `required('image!name')` function).
+     * resource (which should be wrapped in the `require('image!name')` function).
```

Comment:
Agree, but let's test it internally first. I'd say in a week or so. We should still support using images that are managed by Xcode.